src/rpc/util.h

```diff
@@ -425,8 +423,9 @@ class RPCHelpMan
      * @see MaybeArg for handling optional arguments without default values.
      */
     template <typename R>
-    auto Arg(size_t i) const
```

Comment:
nit: there's still a remaining reference to `Arg<Type>(i)` in `CheckRequiredOrDefault`:

<details>
<summary>git diff on fadb3eb57b</summary>

```diff
diff --git a/src/rpc/util.cpp b/src/rpc/util.cpp
index f683878054..d4079d9f37 100644
--- a/src/rpc/util.cpp
+++ b/src/rpc/util.cpp
@@ -673,7 +673,7 @@ static const UniValue* DetailMaybeArg(CheckFn* check, const std::vector<RPCArg>&
 
 static void CheckRequiredOrDefault(const RPCArg& param)
 {
-    // Must use `Arg<Type>(i)` to get the argument or its default value.
+    // Must use `Arg<Type>` to get the argument or its default value.
     const bool required{
         std::holds_alternative<RPCArg::Optional>(param.m_fallback) && RPCArg::Optional::NO == std::get<RPCArg::Optional>(param.m_fallback),
     };

```
</details>